packages/muelu/src/Graph/UncoupledAggregation/MueLu_UncoupledAggregationFactory_def.hpp

```diff
@@ -70,6 +71,7 @@
 #include "MueLu_Aggregates.hpp"
 #include "MueLu_MasterList.hpp"
 #include "MueLu_Monitor.hpp"
+#include "MueLu_FactoryManager.hpp"
```

Comment:
This file is also already included by "MueLu_UncoupledAggreationFactory_decl.hpp".